crimsobot/cogs/reactions.py

```diff
@@ -0,0 +1,183 @@
+        # check if owner for server-specific override
+        owner = ctx.author.id in ADMIN_USER_IDS
+
+        try:
+            fact_object = await FunFacts.get_by_id(fact_id, ctx.guild.id, owner)
+        except DoesNotExist:
+            embed = c.crimbed(title=None, descr=f'Fact {fact_id} does not exist (at least not in this server)!')
+            await ctx.send(embed=embed, delete_after=18)
+            return
+
+        fact_adder = self.bot.get_user(fact_object.created_by.discord_user_id)
+
+        guild = self.bot.get_guild(fact_object.created_in)
+
+        embed = c.crimbed(
+            title=f'FACT INSPECT // ID: {fact_object.uuid}' + ' (admin view)' if owner else '',
+            descr=None,
+            footer='Users with Manage Messages permission can remove facts using ">fact remove [ID]"',
+            thumb_name='think',
+            color_name='yellow',
+        )
+
+        field_list = [
+            ('Subject', fact_object.subject),
+            ('Body', fact_object.funfact),
+            ('Added by', f'{fact_adder} ({fact_adder.id})'),
+            ('Server', f'{guild.name} ({guild.id})'),
+            (
+                'Added on',
+                '{d.year}-{d.month:02d}-{d.day:02d} {d.hour:02d}:{d.minute:02d}:{d.second:02d}'.format(
+                    d=fact_object.created_at)
+            ),
+        ]
+
+        for field in field_list:
+            embed.add_field(name=field[0], value=field[1], inline=False)
+
+        await ctx.send(embed=embed)
+
+    @fact.command(name='remove', aliases=['delete'], brief='Remove a fact by ID.')
+    @has_guild_permissions(manage_messages=True)
+    async def remove_fact(self, ctx: commands.Context, fact_id: int) -> None:
+        """Remove a fact by its ID. To use this command, you must have Manage Messages permission."""
+        # check if owner for server-specific override
+        owner = ctx.author.id in ADMIN_USER_IDS
+
+        fact_removed = await FunFacts.delete_by_id(fact_id, ctx.guild.id, owner)
+
+        if fact_removed != 0:
+            embed = c.crimbed(title=None, descr=f'Fact {fact_id} removed!')
+        else:
+            embed = c.crimbed(
+                title=None,
+                descr='This fact was not removed!',
+                footer='· Did you input the correct fact ID?\n· Was this fact created in this server?',
+            )
+
+        await ctx.send(embed=embed, delete_after=18)
+
+    @fact.command(name='wipe', aliases=['remove_subject', 'delete_subject'],
+                  brief='Remove all facts of a given subject')
+    @has_guild_permissions(manage_messages=True)
+    async def remove_subject(self, ctx: commands.Context, *, subject: CleanMentions) -> None:
+        """Remove all facts in a server with the same subject. Useful for removing spam."""
+        facts_removed = await FunFacts.delete_by_subject(subject, ctx.guild.id)
+
+        if facts_removed != 0:
```

Comment:
Any number that is not zero is a truthy value, so you can do something like
```suggestion
        if facts_removed:
```
instead.